doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -33,261 +33,488 @@ Use the :ref:`matter_template_sample` sample as the base for building a sensor d
-         * ``<include>`` - Defines a cluster that should be included in the device.
-
-            * ``cluster`` - The name of the cluster.
-            * ``client`` - Indicates if the client role should be enabled.
-            * ``server`` - Indicates if the server role should be enabled.
-            * ``clientLocked`` - Indicates if the client role modification should be locked.
-            * ``serverLocked`` - Indicates if the server role modification should be locked.
-            * ``<requireAttribute>`` - Indicates a required attribute's define.
-            * ``<requireCommand>`` - Indicates a required command's define.
-
-      For example, the following XML code defines a structure with one item of type octet string and length 128:
-
-      .. code-block:: xml
-
-         <deviceType>
-            <name>my-new-device</name>
-            <domain>CHIP</domain>
-            <typeName>My new device</typeName>
-            <profileId editable="false">0x0FFF</profileId>
-            <deviceId editable="false">0x001</deviceId>
-            <class>Simple</class>
-            <scope>Endpoint</scope>
-            <clusters lockOthers="true">
-            <include cluster="MyNewCluster" client="true" server="true" clientLocked="false" serverLocked="false"/>
-               <requireAttribute>MY_ATTRIBUTE</requireAttribute>
-               <requireCommand>MyCommand</requireCommand>
-            </clusters>
-         </deviceType>
+   .. tab:: Using the nRF Connect Matter Manufacturer Cluster Editor
+
+      Before using the tool, you need to download its preview version and unpack it on your machine.
+      First, make sure you have completed the steps listed in the :ref:`ug_nrfconnect_manufacturer_cluster_editor_tool_downloading_installing` user guide.
+
+      Once you have the tool installed, you can create a new cluster description file.
+      Complete the following steps:
+
+      1. Edit the :guilabel:`Cluster` tab contents.
+
+         a. Open the :guilabel:`Cluster` tab.
+         #. Fill in the domain, name, code, define, and description of the cluster as follows:
+
+            .. figure:: images/matter_creating_custom_cluster_cluster_page.png
+               :alt: Cluster tab
+
+      #. Add a new command in the :guilabel:`Commands` tab.
+
+         a. Open the :guilabel:`Commands` tab.
+         #. Click :guilabel:`Add command` to open edit box.
+         #. In the edit box, set the following values:
+
+            * **Name** as ``MyCommand``
+            * **Code** as ``0xFFF10000``
+            * **Source** as ``client``
+            * **Response** as ``MyCommandResponse``
+            * **Description** as ``Command that takes two uint8 arguments and returns their sum``
+
+         #. Click :guilabel:`Arguments`.
+         #. In the new edit box, click the plus icon to create a new argument.
+         #. Fill in :guilabel:`Name` as ``arg1``, :guilabel:`Type` as ``int8u``.
```

Comment:
In the description there is ``int8u``, but the picture shows ``uint8``. Can we make it consistent (probably easier to change it in text to match the picture).

Reply:
right, I will fix it.